dashboard/src/main/java/no/sikt/nva/monitoring/UpdateDashboardHandler.java

```diff
@@ -0,0 +1,29 @@
+package no.sikt.nva.monitoring;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.CloudFormationCustomResourceEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class UpdateDashboardHandler implements RequestHandler<CloudFormationCustomResourceEvent, Void> {
+
+    private static final Logger logger = LoggerFactory.getLogger(UpdateDashboardHandler.class);
+
+    public UpdateDashboardHandler() {
+
+    }
+
+    @Override
+    public Void handleRequest(CloudFormationCustomResourceEvent cloudFormationCustomResourceEvent, Context context) {
+        logger.info("Trigges ved stack updates");
+        //Future plans:
+        //List all lambdas in the account
+        //List all alarms in the account
+        //Create cloudwatch widgets for lambdas
```

Comment:
I guess all lambdas could be a issue, so probably some work figuring out how to group, filter or split this?

Reply:
When clicking in aws console, this was not a problem. But I guess there is one way to find out :)